docs/book/middleware.md

```diff
@@ -5,41 +5,81 @@ for your message data, you have to convert this data to an array before you can
 > Note: The middleware uses an array for the message data
 
 ## CommandMiddleware
-The `CommandMiddleware` dispatches the message data to the command bus system. This middleware needs an request attribute 
-(`$request->getAttribute(\Prooph\HttpMiddleware\CommandMiddleware::NAME_ATTRIBUTE)`) called `prooph_command_name`. 
-This name is used for the `\Prooph\Common\Messaging\MessageFactory` to create the `\Prooph\Common\Messaging\Message` 
-object. The data for the command is extracted from the body of the request (`$request->getParsedBody()`) and must be an 
+The `CommandMiddleware` dispatches the message data to the command bus system. This middleware needs an request attribute
+(`$request->getAttribute(\Prooph\HttpMiddleware\CommandMiddleware::NAME_ATTRIBUTE)`) called `prooph_command_name`.
+This name is used for the `\Prooph\Common\Messaging\MessageFactory` to create the `\Prooph\Common\Messaging\Message`
+object. The data for the command is extracted from the body of the request (`$request->getParsedBody()`) and must be an
 array.
 
 ## QueryMiddleware
-The `QueryMiddleware` dispatches the message data to the query bus system. This middleware needs an request attribute 
-(`$request->getAttribute(\Prooph\HttpMiddleware\QueryMiddleware::NAME_ATTRIBUTE)`) called `prooph_query_name`. 
-This name is used for the `\Prooph\Common\Messaging\MessageFactory` to create the `\Prooph\Common\Messaging\Message` 
-object. The data for the query is extracted from query params of the request (`$request->getQueryParams()`) and must be an 
-array.
+The `QueryMiddleware` dispatches the message data to the query bus system. Unlike the other middleware, the
+`QueryMiddleware` supports sending multiple messages using a *POST* request.
+
+### Using GET
+
+Using *GET*, the `QueryMiddleware` only supports a single message. The `prooph_query_name` attribute must be present as
+an `attribute` on the request. Any additional query parameters will be added to the body of the message as well. An
+example URL template would be
+
+`GET /query/{prooph_query_name}`
```

Comment:
docs needs to be changed here, no more GET

Reply:
Thanks for the catch. Updated the docs.